infra/app/database.bicep

```diff
@@ -60,6 +63,11 @@ var containers = [
           type: 'diskANN'
         }
       ]
+      fullTextIndexes: [
+        {
+          path: '/tags'
```

Comment:
Should we leave the index on description too? We aren't sure what will end up working best so maybe leaving both description and tags gives us more properties to test the flow out on. It's hard to know for sure until we can test a few queries

Reply:
Added /description as well.